application/src/main/java/org/wecancodeit/serverside/model/Comment.java

```diff
@@ -1,4 +1,33 @@
 package org.wecancodeit.serverside.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Collection;
+@Entity
 public class Comment {
+
+    @Id
+    @GeneratedValue
+    private String commentName;
+
+    @ManyToMany
+    private Collection<Comment> comments;
+
+    @ManyToMany
+    private Collection<Post> posts;
+
+    @ManyToMany
```

Comment:
If this is included, it should also be ManyToOne for the same reason, but this probably isn't needed. If we want to allow comments on events, that would likely need to be an entirely separate class